pages/attendee/badgedex.js

```
@@ -38,11 +30,25 @@ function BadgeButton({ text, val, setValue, selected }) {
 }
 
 function Badgedex() {
-  let badges = [];
-  for (var i = 0; i < 18; i++) {
-    badges.push(<Badge />);
-  }
-  const [val, setValue] = useState(true);
+  const [allBadges, updateAllBadges] = useState([]);
+  const [all, updateAll] = useState(true);
+  const [filter, updateFilter] = useState(null);
+
+  const { attendee } = useAuth();
+
+  useEffect(() => requestBadges(), [allBadges]);
+  const requestBadges = () => {
+    getAllBadges()
+      .then((response) => updateAllBadges(response.data))
+      .catch();
```

You should give some user feedback. Or in worst case, let it crash. Having a catch and not really handle the error, only make it fail silently.